src/engine/execute.js

```diff
@@ -52,7 +53,7 @@ var execute = function (sequencer, thread) {
 
 
     if (!opcode) {
-        console.warn('Could not get opcode for block: ' + currentBlockId);
+        log.warn('Could not get opcode for block: ' + currentBlockId);
```

Comment:
not sure if I'm a fan of having to import log to do console things, but I guess it makes sense for final linted commits.